bot/responses/sending_videos/compile_clips_handler_responses.py

```diff
@@ -29,6 +29,8 @@ def get_log_no_previous_search_results_message() -> str:
 def get_log_no_matching_segments_found_message() -> str:
     return "No matching segments found for compilation."
 
+def get_log_compiled_clip_is_too_long_message(username: str) -> str:
```

Comment:
czemu tu są jezyki pomieszane?

Reply:
w sensie ? po polsku jest komunikat dla usera a po angielsku nasze logi czy to do bazy czy po prostu na konsole